.tekton/build-nodejs-ex.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: shipwright.io/v1alpha1
+kind: Build
```

Comment:
Does this imply that PaC needs RBAC to create Build objects?

Reply:
This is not used so far as even with https://github.com/openshift-pipelines/pipelines-as-code/pull/656, pac won't apply this type. *But* yes, if PaC was apply it, we would need the rbac to do it. *And* if we use embedded spec (if supported), I think that would be similar.